### PR TITLE
Hold cancels

### PR DESCRIPTION
**Pull request by cdmo**
#8
#5
#61

### CODE REVIEW

**Comment by cdmo** [comment]
Will handle https://github.com/psu-libraries/myaccount/issues/61 too

**Comment by cdmo** [comment]
Adding jest and eslint as well, now that there's javascript

**Comment by cdmo** [comment]
There is more work left to do for holds. But I'd like to submit this as a PR and continue the remaining work in a separate branch. The reason is mainly because this branch already has a lot of weight. Will create remaining issues for tasks. That okay?

**Comment by cdmo** [comment]
I don't know why it's not generating an "Overall coverage" thing, but I am seeing all the correct files in [the latest test](https://codeclimate.com/repos/5e14976585435f212800b3f9/settings/test_reports/5e1e17e90e0d455fd5000a6b)

**Comment by cdmo** [comment]
Oh and now it does, just took it a minute I guess

**Comment by cdmo** [comment]
[Code Climate docs](https://docs.codeclimate.com/docs/travis-ci-test-coverage#section-travis-ci-single-test-suite-non-parallel-builds). Used example 1 from multiple test suites.

**Comment by banukutlu** [comment]
> Code Climate docs

Thanks, this is helpful :)

File: app/javascript/select_all/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+          </th>
+          <td>Pattee Library and Paterno Library Stacks</td>
+          <td>Never expires</td>
+        </tr>
+    </tbody>
+  </table>
+  ```
+  Which yields:
+  
+  ![Screen recording of select all checkbox in acttion](./checked.gif)
```
Comment (banukutlu) [inline app/javascript/select_all/README.md]: 😎

File: app/javascript/select_all/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const selectAll = {
+    start() {
+        const targetKeywords = this.extractCheckboxTargets();
+        let that = this;
```
Comment (banukutlu) [inline app/javascript/select_all/index.js]: 👌

File: app/javascript/select_all/README.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+# Select All
```
Comment (banukutlu) [inline app/javascript/select_all/README.md]: very nice doc 👌

Comment (cdmo) [reply app/javascript/select_all/README.md]: 👍

File: app/javascript/select_all/README.md (hunk 1 above)
Comment (cdmo) [reply app/javascript/select_all/README.md]: Screen recording = crucial 😆

Comment (banukutlu) [reply app/javascript/select_all/README.md]: :100:

**Pull request merged**